doc/classes/ProjectSettings.xml

```diff
@@ -2316,6 +2319,9 @@
 		<member name="physics/3d/run_on_separate_thread" type="bool" setter="" getter="" default="false">
 			If [code]true[/code], the 3D physics server runs on a separate thread, making better use of multi-core CPUs. If [code]false[/code], the 3D physics server runs on the main thread. Running the physics server on a separate thread can increase performance, but restricts API access to only physics process.
 		</member>
+		<member name="physics/3d/run_on_separate_thread.editor" type="bool" setter="" getter="" default="false">
+			Editor-only override for [member physics/3d/run_on_separate_thread]. If [code]true[/code], the 3D editor's physics server runs on a separate thread, making better use of multi-core CPUs, however, certain editor features may not appear smooth on high refresh rate displays due to restricted physics API access.
```

Comment:
Something similar to:
```suggestion
			Editor-only override for [member physics/3d/run_on_separate_thread]. This is disabled in the editor because certain editor features may not appear smooth on high refresh rate displays, due to restricted physics API access.
```
Or:
```suggestion
			Editor-only override for [member physics/3d/run_on_separate_thread]. Enabling this in the editor causes certain certain features to not appear smooth on high refresh rate displays, due to restricted physics API access.
```
Or similar.
Also applies to the 2D equivalent

Reply:
Sweet, updated!